### PR TITLE
Nas 100726

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="form-error" *ngIf="control.dirty || control.touched">
+    <mat-error *ngIf="control.hasError('required')">
+        {{ config.placeholder }} {{ " is required" | translate }}
+    </mat-error>
+    <mat-error *ngIf="control.hasError('atLeastOne')">
+        {{control.errors.fields[0] + " and " + control.errors.fields[1] + ". At leaset one is required" | translate }}
```
Comment (erinix) [inline src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html]: you misspelled "least"

**Comment by erinix** [comment]
You should also style these similar to how Dennis styled mat-error, that way there isn't a huge gap between the input field and the error message

**Comment by erinix** [comment]
@freenas/docs-team could you review the actual messages in this PR

File: src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div class="form-error" *ngIf="control.dirty || control.touched">
+    <mat-error *ngIf="control.hasError('required')">
+        {{config.placeholder}} {{" is required." | translate}}
+    </mat-error>
+    <mat-error *ngIf="control.hasError('atLeastOne')">
+        {{control.errors.fields[0]}} {{" and " | translate}} {{control.errors.fields[1]}} {{". At least one is required." | translate}}
+    </mat-error>
+    <mat-error *ngIf="control.hasError('greaterThan')">
+        {{"Value should be greater than " | translate}} {{control.errors.fields[0]}}.
```
Comment (wblock) [inline src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html]:
```suggestion
        {{"Value must be greater than " | translate}} {{control.errors.fields[0]}}.
```

File: src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    </mat-error>
+    <mat-error *ngIf="control.hasError('forbidden')">
+        {{"This name is in used." | translate}}
```
Comment (wblock) [inline src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html]:
```suggestion
        {{"This name has already been used." | translate}}
```

File: src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    </mat-error>
+    <mat-error *ngIf="control.hasError('matchOther')">
+        {{"The passwords do not match." | translate}}
+    </mat-error>
+    <mat-error *ngIf="control.hasError('range')">
+        {{"The value is out of range, enter a value between " | translate}} {{control.errors.rangeValue.min}} {{" and " | translate }} {{control.errors.rangeValue.max}}.
```
Comment (wblock) [inline src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html]:
```suggestion
        {{"The value is out of range. Enter a value between " | translate}} {{control.errors.rangeValue.min}} {{" and " | translate }} {{control.errors.rangeValue.max}}.
```

File: src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    </mat-error>
+    <mat-error *ngIf="control.hasError('regex')">
+        {{"Invalid format or include invalid character." | translate}}
```
Comment (wblock) [inline src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html]:
```suggestion
        {{"Invalid format or character." | translate}}
```

**Pull request merged**